assembly/src/main/resources/kubernetes.cfg

```diff
@@ -0,0 +1,5 @@
+#
+# Label selector used to idenfity Cellar nodes in Kubernetes cluster
+#
+pod.label.key = name
+pod.label.value = cellar
```

Comment:
Tbh, it is not. I don't even understand what are the implication of not having a newline. Can you please explain?